joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonTypeRecovery.scala

```diff
@@ -57,7 +57,16 @@ class RecoverForPythonFile(
   override def visitImport(importCall: Call): Unit = {
     importCall.argument.l match {
       case (path: Literal) :: (funcOrModule: Literal) :: alias =>
-        val calleeNames = extractPossibleCalleeNames(path.code, funcOrModule.code)
+        val tpath = if (path.code == ".") {
+          // TODO: pysrc2cpg does not link files to the correct namespace nodes
+          val root     = cpg.metaData.root.headOption.getOrElse("")
+          val fileName = path.file.name.head.stripPrefix(root)
```

Comment:
Use `headOption` here and handle case where `path.file.name` is empty.

Reply:
done